src/k8s/cmd/k8s/k8s_x_list_images.go

```diff
@@ -0,0 +1,21 @@
+package k8s
+
+import (
+	"strings"
+
+	cmdutil "github.com/canonical/k8s/cmd/util"
+	"github.com/canonical/k8s/pkg/k8sd/images"
+	"github.com/spf13/cobra"
+)
+
+func newXListImagesCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
+	cmd := &cobra.Command{
+		Hidden: true,
+		Use:    "x-list-images",
```

Comment:
Why is this called "x-....". Sorry I am not familiar with this concept.

Reply:
we can also call this `k8s list-images`. it is a hidden command either way, so it would not matter